src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts

```diff
@@ -82,11 +82,15 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
   private rerenderDataTable(): void {
     if (this.dtElement && this.dtElement.dtInstance) {
       this.dtElement.dtInstance.then((dtInstance: DataTables.Api) => {
```

Comment:
I think it is not a correct way to implement a try catch, you can use the if / else or else try / catch but not the two together.

I am attaching a link with information on the correct use of a try / catch.
https://javascript.plainenglish.io/the-dark-side-of-typescript-try-catch-deeded18ba0d